src/book/views.py

```diff
@@ -33,7 +33,18 @@ def form_valid(self, form):
     def get_success_url(self):
         return reverse_lazy('book:book-detail', kwargs={'pk': self.object.book.pk})
 
-
+class ShowComments(BaseTemplatePageMixin, generic.ListView):
```

Comment:
Please also add pagination support something like showing only 5 comments with page. https://docs.djangoproject.com/en/dev/topics/pagination/#paginating-a-list-view

In general when you show the list data pagination should be applied all the time. Because if you will have big amount of data like 10000 comments in the system and showing it all on the same page browser will crash and do not show anything

Reply:
fixed